sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -461,9 +480,41 @@ export class ApplicationControllerService {
       applicationEndDate: application.currentAssessment?.offering?.studyEndDate,
       applicationInstitutionName:
         application.location?.institution.legalOperatingName,
+      changes,
     };
   }
 
+  /**
+   * Recursively converts the {@link ApplicationDataChange} service model to the
+   * DTO model {@link ApplicationDataChangeAPIOutDTO} which will ensure
+   * that only required properties will be returned from the API also preventing
+   * that future changes in the service model will not be directly returned.
+   * @param applicationDataChanges service model application changes.
+   * @param applicationDataChangeAPIOutDTO converted API DTO model.
+   */
+  transformToApplicationChangesDTO(
+    applicationDataChanges: ApplicationDataChange[],
+    applicationDataChangeAPIOutDTO: ApplicationDataChangeAPIOutDTO[],
+  ) {
```

Comment:
return void

Reply:
Is `void` enforced along the PRs? I never saw it as mandatory, but may be my misunderstanding. I can make the change and start enforcing during the PR reviews since ii has a comment and a thumbs up.